src/configsys.c

```diff
@@ -262,7 +262,7 @@ gint config_setint (const gchar *key, const glong val)
     return 0;
 }
 
-gint config_setnint(const gchar *key, const glong val, const guint idx)
+gint config_setnint (const gchar *key, const glong val, const guint idx)
```

Comment:
Refactorings and whitespace changes should go into a separate commit, and preferably a separate pull request.